delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/package.scala

```diff
@@ -0,0 +1,7 @@
+package ch.epfl.bluebrain.nexus.delta.rdf
+
+import monix.bio.IO
+
+package object jsonld {
+  type IOErrorOr[A] = IO[JsonLdError, A]
```

Comment:
There are a lot of aliases, maybe it would be better to use the actual type to avoid having to look up their meaning. Thoughts?

Reply:
Most of the times this one is used `IOErrorOr[A]`, since in most parts of the code I deal with `JsonLdError`.